runtime/queries/ada/textobjects.scm

```diff
@@ -0,0 +1,23 @@
+;; Support for high-level text objects selections.
+;; For instance:
+;;    vaf     (v)isually select (a) (f)unction or subprogram
+;;    vif     (v)isually select (i)nside a (f)unction or subprogram
+;;    vai     (v)isually select (a) (i)f statement (or loop)
+;;    vii     (v)isually select (i)nside an (i)f statement (or loop)
+;;
+;; https://github.com/nvim-treesitter/nvim-treesitter-textobjects/blob/master/README.md
```

Comment:
We can link to the textobjects guide instead here: https://docs.helix-editor.com/master/guides/textobject.html

And https://docs.helix-editor.com/master/usage.html#navigating-using-tree-sitter-textobjects